### ISSUE TITLE
FLOPs

### ISSUE THREAD

**Issue by Cydia2018** [kssteven418/LTP#6]
Since it is a dynamic transformer, the GFLOPs of each instance input is different. How to calculate the FLOPs of the entire model? Take the average FLOPs of all validation sets?

**Comment by kssteven418** [comment]
Yes, the current implementation counts the averaged FLOPs across all the examples in the validation set

**Comment by Cydia2018** [comment]
> Yes, the current implementation counts the averaged FLOPs across all the examples in the validation set

Thanks for your reply!
